Add concurrently to run server and client via one start:dev command (#38)
cuttle-cards/cuttle: README.md

```
**NOTE** `ci` (as opposed to `install`) is a 'clean install' which ensures versions exactly match package-lock.json).

> #### Boot the App

Reviewer: While we're at it, can we say 'Start' instead of 'Boot' in these headers? I think aligning the language with the command names could help make them easier to remember

> #### Boot the App

Author: ```suggestion
#### Start the App
```

> You can run back the server and client with one command to start the client on localhost:8080.

Author: ```suggestion
You can simultaneously run the server on `localhost:1337` and the client on `localhost:8080` with one command:
```

```
npm run start:dev
```

Alternatively, you can run them independantly with separate commands.

> ##### Boot the Server (sails backend)

Author: ```suggestion
##### Start the Server (sails backend)
```

```
npm run start:server
```

> ##### Boot the Client (Vue SPA)

Author: ```suggestion
##### Start the Client (Vue SPA)
```
